runtime/common/src/benchmarking/orml_asset_registry.rs

```diff
@@ -76,14 +78,14 @@ pub mod benchmarks {
 			RawOrigin::Root,
 			CurrencyId::Native,
 			Some(123),
-			Some(vec![b'b', 128]),
-			Some(vec![b'b', 128]),
+			Some(BoundedVec::truncate_from(vec![b'b', 128])),
```

Comment:
So, the reason for not using `longest_vec::<T>()` here, is the change from `b'a'` to `b'b'` ?

Reply:
I assumed we want to update to something different, `longest_vec` would not test the actual modification since it will return the same vec.